yml/cdb_latiss.yaml

```diff
@@ -914,6 +914,11 @@ tables:
     datatype: int
     description: Count of high signal-to-noise-ratio sources.
     ivoa:unit: count
+  - name: postisr_pixel_median
+    "@id": "#visit1_quicklook.postisr_pixel_median"
+    datatype: float
+    description: Median postISR pixel value.
+    ivoa:unit: count
```

Comment:
If it's a float, I don't think it can be a count.  I thought there had been discussion about the unit being in electrons or some other gain-corrected value.